clowdapp.yaml

```diff
@@ -162,13 +162,13 @@ parameters:
     description: maximum pod replicas for horizontal pod autoscaler
     value: "1"
   - name: CPU_LIMIT
-    value: 500m
+    value: 700m
```

Comment:
Should we set these values in the parameters section (lines 164-171)?  If not, then we should remove the parameters if they're not being used.

Reply:
In this case they are in the parameters :)

Reply:
Oops, yes.  You're right.  Mostly looked at hsp; just quickly looked at baseline and assumed it was the same.